keccak-air/src/constants.rs

```diff
@@ -8,7 +8,7 @@ pub(crate) const R: [[u8; 5]; 5] = [
     [27, 20, 39, 8, 14],
 ];
 
-const RC: [u64; 24] = [
+pub const RC: [u64; 24] = [
```

Comment:
It looks like SP1 uses this to basically implement native Keccak execution? Feels a bit more natural to me to have that whole function in `keccak-air`, but no need to change anything now.